docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -106,17 +111,111 @@ $ pip install -r requirements-dev.txt
 $ pip install setup.py
 ```
 
+After installation, ``xrdsst`` command runs the toolkit, when invoked without any parameters,
+it will give the overview of available options and sub-commands. Sub-commands themselves can
+also have further subcommands, so for example all the supported token operations can be listed
+with ``xrdsst token``:
+
+```
+$ xrdsst token
+usage: xrdsst token [-h] [-v] {init-keys,list,login} ...
+```
+
+
 ## 3 Configuration of X-Road Security Server
 
 ### 3.1 Prerequisites to Configuration
 
-* a central server running the Ubuntu 18.04 LTS or 20.04 LTS operating system, on an x86-64bit platform. 
-* a security server providing management services. 
-* single or multiple security servers(to be configured by the X-Road Security Server Toolkit) running the Ubuntu version 18.04 LTS or 20.04 LTS 
-  or Redhat version 7 or 8 operating system, on an x86-64bit platform or X-Road Security Server Sidecar running in a Docker container
-* configuration file in YAML format for configuring security server
+* X-Road central server. In development, this is best run in the mode where auto-approvals
+are enabled, to be able to register authentication certificates and manage security server
+clients without taking separate actions at central server, see [UG-CS](#Ref_CS-UG) about
+``auto-approve-auth-cert-reg-requests`` and ``auto-approve-client-reg-requests``.
```

Comment:
I would leave out the 'in development' since the settings can also be enabled in production. Perhaps just say that the toolkit works the smoothest when the auto approvals are enabled.

Reply:
I would rather leave it like this at the moment, since I am not yet convinced that there are ever going to be any production central servers with policies that would allow enabling these auto-approvals.